src/apps/loyalty/server/route_handlers.tsx

```diff
@@ -38,7 +42,7 @@ export function ThankYou(req: Request, res: Response, next: NextFunction) {
   let html
 
   if (info.loyalty_applicant_at) {
```

Comment:
So basically, if you are already a loyalty applicant, we just show you the 'Repeat Visitor' page, unless you have the special query param that indicates you just submitted the application, in which case you'd get the appropriate 'Thank You' page.